src/controller/CHIPCluster.h

```diff
@@ -51,6 +51,7 @@ class DLL_EXPORT ClusterBase
     virtual ~ClusterBase() {}
 
     CHIP_ERROR Associate(DeviceProxy * device, EndpointId endpoint);
+    CHIP_ERROR Associate(GroupId groupId, DeviceProxy * device);
```

Comment:
This should probably have a comment about how the order needs to be different because otherwise the functions are ambiguous.

Or call the new one `AssociateWithGroup`?

Reply:
Renaming the function will prevent ambiguity. Done